polyfill/ReflectionEnumPolyfill.php

```diff
@@ -5,10 +5,27 @@
 }
 
 /** @internal */
-class ReflectionEnum
+class ReflectionEnum extends \ReflectionClass
```

Comment:
I decided to drop PHP 8.0 support anyway, which will remove the polyfill entirely.

That said, patch stays open until I'm done with my changes.